_common/qiskit/benchmark_runner.py

```diff
@@ -93,21 +93,20 @@
 
     args = parser.parse_args()
 
-# For Inserting the Noise model default into exec option as it is function call 
+    # For Inserting the Noise model default into exec option as it is function call 
     if args.noise_model != None :
         
         if args.noise_model == 'None':
```

Comment:
I think it's confusing to have two different "None" arguments; one being a string and one being a `NoneType`. I was able to pass in a `None` into the parser and have this be interpreted as `NoneType` so I'm still not entirely following why these two checks need to be done.